meta-ostro/recipes-soletta/soletta/soletta_%.bbappend

```diff
@@ -1,6 +1,6 @@
 FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
 
-DEPENDS_append = " systemd"
+DEPENDS_append = " systemd libmicrohttpd"
```

Comment:
I just added the DEPENDS here to get a good build to try (to make sure the right version gets used).

I'm closely following meta-soletta#48